core/new-gui/src/app/dashboard/service/auth-guard/auth-guard.service.ts

```diff
@@ -0,0 +1,14 @@
+﻿import { Injectable } from "@angular/core";
+import { Router, CanActivate } from "@angular/router";
+import { UserService } from "../../../common/service/user/user.service";
+@Injectable()
+export class AuthGuardService implements CanActivate {
```

Comment:
it seems that this service is not only used by dashboard components but also workspace components (i.e., if logout from anywhere, it should be redirected back to login). 

Shall we put this service under `src/app/common/service/` path?

Reply:
Yes.